### ISSUE TITLE
Installation - usermod: Command not found

### ISSUE THREAD

**Issue by Sunsheep** [pi-hole/pi-hole#3345]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).
---
**Disclaimer:**

I found a error, which is not directly a pihole bug. However I will create this bug report for better error handling while running pihole's installation script.

---
**Expected behaviour:**
Pihole successfully installed.

**Actual behaviour:**
Installation script shows the following error but continues installing.

`/etc/.pihole/automated install/basic-install.sh: Zeile 1892: usermod: Command not found.
`

In conclusion the new database file is not created, which leads to a incomplete install. Furthermore no data will displayed in webview.

**Steps to reproduce:**
Remove `/usr/sbin` from `$PATH` variable.

**Troubleshooting undertaken, and/or other relevant information:**

Call full absolute path to `usermod` or add `/usr/sbin` to path variable by:

`PATH=$PATH:/usr/sbin`

Also, canceling the installation and telling the user to set his path variable correctly helps to avoid subsequent problems and may prevent some bug reports.

**Comment by Sunsheep** [comment]
Duplicate of #3209

Sorry for duplicate.
